warehouse/internal/repo/staging.go

```diff
@@ -147,10 +145,6 @@ func (sf *StagingFiles) Insert(ctx context.Context, stagingFile *model.StagingFi
 	}
 	now := sf.now()
 
-	schemaPayload, err := jsonrs.Marshal(stagingFile.Schema)
```

Comment:
Can we drop `Schema` field from `stagingFile` ?

Reply:
Ok. But instead of saving it as {} in the repositry layer, can we assign it {} in the process API itself?

Reply:
IMO process API should not know these details. Even for getting the schemas, it’s the repository layer that knows the details how to compile the schema. Either from snapshot or from the actual one.